views/js/controller/runtime/testRunner.js

```diff
@@ -234,6 +234,11 @@ define([
                     unansweredCount=(this.testContext.numberItemsSection - this.testContext.numberCompletedSection),
                     flaggedCount=this.testContext.numberFlaggedSection;
 
+                if( this.isCurrentItemAnswered() ){
+                    unansweredCount--;
+                }
+
+                this.getQtiRunner().updateItemApi();
```

Comment:
see with @llecaque 
The goal was to fix the code from @bursonic : I just had the time to add the listener of responsechange and stateready but cannot find a way to properly remove the call to this.getQtiRunner().updateItemApi() to submit the response before leaving the item.